Assignment1/style.css

```diff
@@ -0,0 +1,166 @@
+
+    body{
+        margin:0px;
+        padding:0px;
+    }
+   .row{
+       height:80px;
+       text-align:center;
+       background-color:white;
+      
+    }
+   .tableHeader{
+       padding:10px;
+       background-color: white;
+       width:350px;
+       text-align:center;
+   }
+.footer{
+    
+        background-color:tan ;
+    width:100%;
+    height: 30px;
+}
+   .profileTable{
+       width:1024px;
+   }
+  .head{
+      letter-spacing: 2px;
+      font-size:25px;
+  }
+   .profile{
+    padding-top:25px;
+       width:341px;
+       font-size:25px;
+       text-align: center;
+  display: table-cell;
+
+       
+   }
+    .name{
+        color:rgb(82, 80, 80);
+        margin-top:70px;
+        font-size: 25px;
+       letter-spacing: 3px;
+       font-family:sans-serif;
+    }
+    .data{
+        border:2px solid rgb(82, 80, 80);
+    }
+    .image{
+    border-radius:50%;
+    height:150px;
+     width:150px;
+     
+    }
+    .profileImage{
+        display:inline-block;
+        width:341px;
+      
+    }
+    .imageRow{
+        text-align:center;
+    }
+    .designation{
+        color:grey;
+        font-size:15px;
+        letter-spacing: 2px;
+    }
+    .description{
+        color:grey;
+        font-size:20px;
+    text-align: justify;
+         max-height: 320px;
+  position: relative;
+  overflow: hidden;
+    }
+    .description1{
+        color:black;
+        font-size:17px;
+    text-align: left;
+  
+    }
+    .container{
+        width:1024px;
+        margin: 0 auto;
+    }
+    .container1{
+      
+        width:1024px;
+        height:200px;
+        margin:0 auto;
+        background-color:rgb(171, 184, 184);
+    }
+    .namingData{
+        padding-top:48px;
+    }
+    .inner-container{
```

Comment:
at one place you are using camel case and another place you are using - in class name. Why?